stock_reserve_rule/models/stock_reserve_rule.py

```diff
@@ -152,6 +154,21 @@ class StockReserveRuleRemoval(models.Model):
         "When empty, any packaging can be removed.",
     )
 
+    reserve_lot = fields.Selection(
+        selection=[
+            ("lot_qty_gt_reserved_qty", "Lot qty > reserved qty"),
+            ("lot_qty_gte_reserved_qty", "Lot qty >= reserved qty"),
+            ("lot_qty_eq_reserved_qty", "Lot qty = reserved qty"),
+        ],
+        string="Reserve lot if",
+    )
```

Comment:
Do you need this complexity? Or you can always apply lot_qty_gte_reserved_qty ?

Reply:
@jbaudoux we need these options because there are different scenarios

Reply:
@jbaudoux can we mark this as resolved?